nemo/collections/nlp/models/language_modeling/megatron/gpt_model.py

```diff
@@ -167,6 +167,7 @@ def __init__(
         ub_tp_comm_overlap=False,
         use_flash_attention=False,
         seq_len_interpolation_factor=None,
+        base=10000,
```

Comment:
Give it better name - rotary_base or base_positional_scale or something

Reply:
Done. Changed to `rotary_base`